server.js

```diff
@@ -18,9 +35,41 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Home page
+const listEndpoints = require('express-list-endpoints')
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(listEndpoints(app))
+})
+
+// Get thoughts
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
+
+// Post thoughts
+app.post('/thoughts', async (req, res) => {
+  const { message } = req.body
+  const thought = new Thought({ message })
+
+  try {
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+  } catch (err) {
+    res.status(400).json({ message: 'Could not save message to the Database', error: err.errors })
+  }
+})
+
+// Post like to a thought 
+app.post('/thoughts/:thoughtId/like', async (req, res) => {
```

Comment:
Good that you named the endpoints consistently! According to documentation this should be called `/:thoughtId/like` though